src/index.html

```diff
@@ -5,9 +5,44 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <!-- <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com" crossorigin> -->
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="card">
+      <img
+      src="./images/imac.jpeg"
+      alt="imac"
+      class="card__image"
+      >
```

Comment:
```suggestion
      <img
        src="./images/imac.jpeg"
        alt="imac"
        class="card__image"
      >
```
[Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 14 - fix all similar cases